src/Components/StepOne/StepOne.js

```diff
@@ -0,0 +1,45 @@
+import React, {useState} from 'react';
+import './StepOne.scss';
+import DatePicker from 'react-datepicker'
+import 'react-datepicker/dist/react-datepicker.css'
+
+const StepOne = ({nextStep}) => {
+
+    const [startDate, setStartDate] = useState(new Date())
```

Comment:
Using new date here it is gonna pass different timestamp on different renders, and can confuse the user since than that field would be valid, even if he did not have any interaction with that field, is that a requirement?